dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -136,10 +136,6 @@
     @ApiModelProperty(value = "This is the email of the git organization", position = 6)
     private String email;
 
-    @Column
```

Comment:
Removing this might break the CLI. Can you confirm if this is the case or not?
(and how broken or not it is)

Reply:
CLI and the UI is based on the swagger.yaml . I made sure to not alter it. It seems unlikely to break.